### PR TITLE
Fix label tag matching in sidebar

### PR DESCRIPTION
**Pull request by benjaminpkane**
Fixes label tag matching in the sidebar

### CODE REVIEW

File: fiftyone/server/view.py [hunk 1]
```diff
@@ -698,21 +701,22 @@ def _get_filtered_path(view, path, filtered_fields, label_tags):
     return "___%s" % path
 
 
-def _add_frame_labels_tags(path, view):
+def _add_frame_labels_tags(path, field, view):
     path = path[len("frames.") :]
+    items = "%s.%s" % (path, field.document_type._LABEL_LIST_FIELD)
```
Comment (manivoxel51) [inline fiftyone/server/view.py:706]: this is breaking when I load a specific dataset
```
ERROR:root:'ListField' object has no attribute 'document_type'
Traceback (most recent call last):
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/decorators.py", line 34, in wrapper
    response = await func(endpoint, request, data, *args)
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/routes/samples.py", line 29, in post
    results = await paginate_samples(
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/samples.py", line 75, in paginate_samples
    view = fosv.get_view(
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 126, in get_view
    view = get_extended_view(
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 221, in get_extended_view
    view = _add_labels_tags_counts(view, filtered_labels, label_tags)
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 275, in _add_labels_tags_counts
    view = add_tags(path, field, view)
  File "/Users/maniaskari/code/fiftyone/fiftyone/server/view.py", line 735, in _add_labels_tags
    items = "%s.%s" % (path, field.document_type._LABEL_LIST_FIELD)
AttributeError: 'ListField' object has no attribute 'document_type'
```

File: fiftyone/server/view.py [hunk 2]
```diff
@@ -727,16 +731,17 @@ def _add_frame_label_tags(path, view):
     return view
 
 
-def _add_labels_tags(path, view):
+def _add_labels_tags(path, field, view):
+    items = "%s.%s" % (path, field.document_type._LABEL_LIST_FIELD)
```
Comment (manivoxel51) [inline fiftyone/server/view.py:735]: same here

File: fiftyone/server/view.py (hunk 1 above)
Comment (benjaminpkane) [reply fiftyone/server/view.py:706]: Will fix, but there are a plethora of issues with raw label lists which looks like the field type here

File: fiftyone/server/utils.py [hunk 3]
```diff
@@ -153,6 +153,14 @@ def _iter_label_fields(schema):
             and issubclass(field_to_check.document_type, fol.Label)
             and not issubclass(field_to_check.document_type, fol._HasLabelList)
         ):
+            parent_path = ".".join(path.split(".")[:-1])
+            parent_field = schema.get(parent_path, None)
+            if parent_field and issubclass(
+                parent_field.document_type, fol._HasLabelList
```
Comment (manivoxel51) [inline fiftyone/server/utils.py:159]: If parent_field is a list, this breaks

**Pull request merged**